content/academy/web_scraping_for_beginners/challenge.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Challenge
+description: Test your knowledge from the previous sections in this course by building an Amazon scraper with Crawlee!
+menuWeight: 1.4
+paths:
+    - web-scraping-for-beginners/challenge
+---
+
+# Challenge
+
+Before moving onto the other courses in the academy, we recommend following along with this section, as it combines everything you've learned in the previous lessons into one cohesive project that helps you prove to yourself that you've thoroughly understood the material.
+
+We recommended that you make sure you've gone through both the [data collection]({{@link web_scraping_for_beginners/data_collection.md}}) [crawing]({{@link web_scraping_for_beginners/crawling.md}}) sections of this course to ensure the smoothest development process.
+
+## [](#learning) Learning 🧠
+
+Before continuing, it is highly recommended to do the following:
+
+- Look over [how to build a crawler in Crawlee](https://crawlee.dev/docs/introduction/first-crawler) and ideally **code along**
+- Read [this short article](https://help.apify.com/en/articles/1829103-request-labels-and-how-to-pass-data-to-other-requests) about **request labels** and [`userData`](https://crawlee.dev/api/core/class/Request#userData) (this will be extremely useful later on)
```

Comment:
The labels article is outdated. Now you can do just `request.label`. Generally, I would not link any code examples from Help, because they're often outdated and even if they aren't, they have shitty code formatting.